Makefile

```diff
@@ -6,6 +6,10 @@
 # Ensure go-extpoints and go-import-subtree are available for go generate
 export PATH := $(GOPATH)/bin:$(PATH)
 
+bootstrap:
```

Comment:
I wonder if something could be conslidated here with bootstrapping: 
https://github.com/taskcluster/taskcluster-worker/blob/master/.travis.yml#L21